domain/src/main/java/edu/nextstep/camp/calculator/Calculator.kt

```diff
@@ -1,11 +1,16 @@
 package edu.nextstep.camp.calculator
 
 object Calculator {
+    private const val DELIMITER = " "
+
     fun calculate(input: String): Int {
-        val list = input.split(" ")
-        return list.subList(1, list.size)
-            .chunked(2)
-            .map { Pair(IntArithmetics.from(it[0]), it[1].toInt()) }
-            .fold(list[0].toInt()) { acc, curr -> curr.first.apply(acc, curr.second) }
+        val words = input.trim().split(DELIMITER)
+        val identity = words[0].toInt()
+        return words.drop(1)
```

Comment:
매직 넘버를 어떻게 없앨 수 있을지, 혹은 누구나 알기 쉽게 의미를 부여할 수 있을지 막막해서 도움을 요청합니다.
고민을 해보아도 정말로 모르겠습니다.

Reply:
```suggestion
        val words = input.trim().split(DELIMITER)
        val identity = words.first().toInt()
        return words.drop(1)
```
first를 활용해주시면 좋을 것 같아요.
drop에 전달되는 숫자는 유지해도 괜찮을 것 같아요.